io/include/detray/io/backend/detail/basic_converter.hpp

```diff
@@ -1,29 +1,29 @@
 /** Detray library, part of the ACTS project (R&D line)
  *
- * (c) 2023-2024 CERN for the benefit of the ACTS project
+ * (c) 2023-2025 CERN for the benefit of the ACTS project
```

Comment:
In Acts core we have stopped doing this, and just use the initial year of the project start.

Reply:
Yes, we can switch to that, too

Reply:
Yes!!!